app/concerns/owable.rb

```diff
@@ -48,7 +48,7 @@ def update_board_cameos
 
       # adjust for the fact that the associations are managed separately
       all_authors = authors + unjoined_authors + joined_authors + tagging_authors
-      new_cameos = all_authors.uniq - board.writers
+      new_cameos = all_authors.uniq - (board.writers + board.cameos)
```

Comment:
Does board.authors include board.creator? Because if it does we can use that and then we don't need to do the addition.

Reply:
board.authors seems to not exist, actually! it's just board.coauthors and board.cameos

Reply:
sorry, I meant board.board_authors

Reply:
Ah. Having poked at Effulgence in my dev env, the creator doesn't seem to have a BoardAuthor object.

Reply:
Blah okay, we'll fix it if we ever get that sorted